package.json

```diff
@@ -20,9 +20,9 @@
     "koa-sendfile": "2.0.0",
     "koa-timeout": "0.1.1",
     "metric-log": "1.0.1",
-    "node-uuid": "1.4.7",
     "opbeat": "3.16.0",
     "s3": "4.4.0",
+    "uuid": "^3.0.0",
```

Comment:
Other package versions seem to pinned to in this repo.

Reply:
after some time... 👍